tests/90jira/SYN-391.pl

```diff
@@ -0,0 +1,30 @@
+multi_test "Setting push rules twice 500s  SYN-391",
+    requires => [qw( register_new_user http_clients do_request_json_for )],
+    do => sub {
+        my ( $register_new_user, $clients, $do_request_json_for ) = @_;
+        my $http = $clients->[0];
+        my $alice;
+
+        Future->needs_all(
+            $register_new_user->($http, "90jira-SYN-391_alice"),
```

Comment:
Any reason to be using `$register_new_user` here? If the requirements on the user aren't anything overly special, just use a `local_user_fixture()` like most of the other tests, to get a new user automatically provisioned.